src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -867,7 +868,10 @@ protected override void DecodeWellKnownAttributeImpl(ref DecodeWellKnownAttribut
             {
                 if (!this.IsValidUnscopedRefAttributeTarget())
                 {
-                    diagnostics.Add(ErrorCode.ERR_UnscopedRefAttributeUnsupportedTarget, arguments.AttributeSyntaxOpt.Location);
+                    var code = !UseUpdatedEscapeRules
+                        ? ErrorCode.WRN_UnscopedRefAttributeOldRules
```

Comment:
>WRN_UnscopedRefAttributeOldRules [](http://example.com/codeflow?start=36&length=32)

Are we reducing an existing error to a warning here, in cases where the previous error applied regardless of which rules were used? For instance, when compiling the following with C# 10 and .NET 6:
```csharp
ref struct R<T> { }
class C
{
    ref T F1<T>([UnscopedRef] R<T> r) => throw null;
    ref T F2<T>([UnscopedRef] ref T r) => throw null;
    ref T F3<T>([UnscopedRef] T r) => throw null;
}
```

Reply:
Yes, I will change that, thanks.